helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -22,6 +22,7 @@ data:
           class denial
           class error
         }
+        loadbalance {{ .Values.loadbalance.policy }}
```

Comment:
Missing `:`

Reply:
where? 

the final config should look like this

```
  Corefile: |
    .:53 {
        errors
        health
        kubernetes cluster.local 172.18.192.0/22 172.18.128.0/18 {
          pods insecure
          upstream
          fallthrough in-addr.arpa ip6.arpa
        }
        prometheus :9153
        proxy . /etc/resolv.conf
        cache 30
        loadbalance round_robin
    }```

Reply:
Yeah, the other colon is just there to prepend the port

Reply:
I misread, all good.